src/script/luna/mememu.cpp

```diff
@@ -1407,6 +1411,13 @@ class NPCMemory final : public NpcParent
         }
 
         NpcParent::setValue(obj, address, value, ftype);
+
+        if(address == 0x3C) // Layer, but setting it is not yet supported.
+        {
+            int index = obj - &NPC[0];
+            if(index >= -128 && index <= maxNPCs)
+                syncLayers_NPC(index);
+        }
```

Comment:
You may just comment out this code with `#if 0` `#endif` block easily until something would appear at any moment is really needed.

Reply:
I will, thanks.